doc/userGuide.md

```diff
@@ -39,15 +39,21 @@ You can set up your own Jenkins-based test builds using the AdoptOpenJDK openjdk
 ``` bash
 git clone https://github.com/AdoptOpenJDK/openjdk-tests.git
 cd openjdk-tests
-get.sh -t openjdk-testsDIR -p platform -v jvmversion [-s downloadBinarySDKDIR] [-r SDK_RESOURCE] [-c CUSTOMIZED_SDK_URL]
+get.sh -t openjdk-testsDIR -p platform [-j SE80] [-i hotspot] [-R latest] [-t jdk] [-s downloadBinarySDKDIR] [-r SDK_RESOURCE] [-c CUSTOMIZED_SDK_URL]
 ```
 
 Where possible values of get.sh script are:
 ```
 Usage : get.sh  --testdir|-t openjdktestdir
                 --platform|-p x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux | ppc64_aix
 
-                --jvmversion|-v openjdk8 | openjdk8-openj9 | openjdk9 | openjdk9-openj9 | openjdk10 | openjdk10-sap
+                [--jdk_vesion|-j ] : optional. JDK version
```

Comment:
spelling: jdk_vesion should be jdk_version

Reply:
Updated. Thanks.